htm/src/CMakeLists.txt

```diff
@@ -251,66 +150,44 @@ set(utils_files
     nupic/utils/StringUtils.cpp
     nupic/utils/StringUtils.hpp
     nupic/utils/VectorHelpers.hpp
-    nupic/utils/Watcher.cpp
-    nupic/utils/Watcher.hpp
+    nupic/utils/Timer.cpp
```

Comment:
Good morning.
> Watcher changed to Timer?
No, Watcher move to engine.   It's an networkAPI thing and the rest of the utils are HTM things.
Timer moved from os to utils because the os things are networkAPI and utils are HTM things.